src/experiment.js

```diff
@@ -3,31 +3,49 @@ const path = require('path');
 const { FILEDIR } = require('./dirs');
 const { authMiddleware } = require('./authentication');
 
+const { timeStamp } = require('./serverUtils');
+
+var Config = require('./models/Config.model');
+
+const collection = 'experiment_data'
+
 module.exports = (router = new Router()) => {
-// Get config file
+  //get all config documents from mongo.
+  router.get('/', authMiddleware);
   router.get('/', (req, res) => {
-    res.sendFile(path.join(FILEDIR, 'config.txt'), {}, (err) => {
-      if (err) {
-        res.status(400).send({
-          message: 'no config',
-        });
-      }
+    Config.find({}).then(function(configs) {
+      var response = {};
+      var count = 1;
+      configs.forEach(function(record) { 
```

Comment:
same note as from `data.js`, more natural to use `map` and to get rid of the Promise.